CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [x.y.z] - YYYY-MM-DD (Unreleased)
 
+### Improvements
+
+- The Grafton client now differentiates between an error and a message returned
+  from the provider, a message will be returned if a provider sends a valid
+  message despite whether or not it was an error response.
+- The Grafton client now takes a `logrus.Entry`, when provided Grafton will
+  push error logs to the provided logger.
```

Comment:
Agreed however we had discussed that we would include changelings for the graft on client along with the command line.